lib/logstash/inputs/mongodb.rb

```diff
@@ -112,7 +112,7 @@ def get_placeholder(sqlitedb, since_table, mongodb, mongo_collection_name)
     if x[:place].nil? || x[:place] == 0
       first_entry_id = init_placeholder(sqlitedb, since_table, mongodb, mongo_collection_name)
       @logger.debug("FIRST ENTRY ID for #{mongo_collection_name} is #{first_entry_id}")
-      return first_entry_id
+      return [first_entry_id, first_entry_id]
```

Comment:
Maybe this should return the placeholder or nil and we can wrap it with a get_processing_start_id() or something that returns us the palceholder or the beginning of the db.